Changelog.md

```diff
@@ -190,6 +190,7 @@ and this project follows to [Ragnarök Versioning Convention](https://gist.githu
 - Deleted broken Hescobastion and UMP45GFL props
 - Removed the vanilla ability to disable shaders with F4 (this allowed for flash to be disabled)
 - Removed the `WIP` status of the flashbang as it is fully usable and reliable
+- Removed the muzzle flash from the VSS Vintorez to make it more similar to it's real-life counterpart
```

Comment:
Should be in the `Fixed` category and be something along the lines of `Fixed VSS Vintorez having a flash when it shouldn't`

Reply:
Did you forgot to push, this is still an issue